### PR TITLE
Export contracts

### PR DESCRIPTION
**Pull request by peterjah**

### CODE REVIEW

File: smart-contracts/assembly/contracts/FT/token-burn.ts [hunk 1]
```diff
@@ -45,16 +40,17 @@ export function burn(binaryArgs: StaticArray<u8>): void {
...
+export function _burn(addressToBurn: Address, amount: u64): void {
   const oldRecipientBalance = _balance(addressToBurn);
   const newRecipientBalance = oldRecipientBalance - amount;
 
   // Check underflow
-  if (oldRecipientBalance < newRecipientBalance) {
-    return false;
-  }
+  assert(
+    oldRecipientBalance > newRecipientBalance,
```
Comment (Thykof) [inline smart-contracts/assembly/contracts/FT/token-burn.ts:49]: opposite of < is >=

```suggestion
    oldRecipientBalance >= newRecipientBalance,
```

**Review by leoloco — APPROVED** [review]

File: smart-contracts/assembly/contracts/FT/token-mint.ts [hunk 2]
```diff
@@ -49,17 +44,17 @@ export function mint(binaryArgs: StaticArray<u8>): void {
...
+function _mint(recipient: Address, amount: u64): void {
   const oldRecipientBalance = _balance(recipient);
   const newRecipientBalance = oldRecipientBalance + amount;
 
-  if (oldRecipientBalance > newRecipientBalance) {
-    return false;
-  }
+  // Check overflow
+  assert(
+    oldRecipientBalance < newRecipientBalance,
```
Comment (Thykof) [inline smart-contracts/assembly/contracts/FT/token-mint.ts:53]: opposite of > is <=

```suggestion
    oldRecipientBalance <= newRecipientBalance,
```

File: smart-contracts/assembly/contracts/FT/token-mint.ts [hunk 3]
```diff
@@ -68,13 +63,15 @@ function _mint(recipient: Address, amount: u64): boolean {
...
   const oldTotalSupply = bytesToU64(totalSupply([]));
   const newTotalSupply = oldTotalSupply + amount;
-  // Overflow
-  if (oldTotalSupply > newTotalSupply) {
-    return false;
-  }
+
+  // Check overflow
+  assert(
+    oldTotalSupply < newTotalSupply,
```
Comment (Thykof) [inline smart-contracts/assembly/contracts/FT/token-mint.ts:72]: opposite of > is <=
```suggestion
    oldTotalSupply <= newTotalSupply,
```

File: smart-contracts/assembly/contracts/FT/token-burn.ts (hunk 1 above)
Comment (peterjah) [reply smart-contracts/assembly/contracts/FT/token-burn.ts:49]: i know but oldRecipientBalance = newRecipientBalance its not an overflow (meaning amount is 0)
IDK any reason for this to happen...

**Review by Thykof — APPROVED** [review]

**Pull request merged**